qdrant-landing/content/blog/qdrant-1.8.x.md

```diff
@@ -0,0 +1,110 @@
+- **User Experience:** Results will appear faster, leading to a smoother experience for the user.
+- **Scalability:** You can easily accomodate rapidly growing users or an expanding dataset.
+
+### Sparse vectors benchmark
+
+Performance results are publicly available for you to test. Qdrant's R&D developed a dedicated [open-source benchmarking tool](https://github.com/qdrant/sparse-vectors-benchmark) just to test sparse vector performance.
+
+A real-life simulation of sparse vector queries was run against the [NeurIPS 2023 dataset](https://big-ann-benchmarks.com/neurips23.html). All tests were done on an 8 CPU machine on Azure. 
+
+Latency (y-axis) has dropped significantly for queries. You can see the before/after here:
+
+![dropping latency](/blog/qdrant-1.8.x/benchmark.png)
+**Figure 1:** Dropping latency in sparse vector search queries across versions 1.7-1.8.
+
+The colors within the scatter plot show the frequency of the results. Frequency is highest where the red dots are. This tells us that latency for sparse vectors queries dropped by a factor of 16. Therefore, the time it takes to retrieve an answer with Qdrant is that much shorter. 
+
+This performance increase can have a dramatic effect on hybrid search implementations. [Read more about how to set this up.](https://qdrant.tech/articles/sparse-vectors/)
+
+FYI, sparse vectors were released in [Qdrant v.1.7.0](https://qdrant.tech/articles/qdrant-1.7.x/#sparse-vectors). They are stored using a different index, so first check out the documentation if you want to try an implementation.
+
+## CPU Resource Management
+
+Indexing is Qdrant’s most resource-intensive process. Now you can account for this by allocating compute use specifically to indexing. You can assign a number CPU resources towards indexing and leave the rest for search. As a result, indexes will build faster, and search quality will remain unaffected.
+
+This isn't mandatory, as Qdrant is by default tuned to strike the right balance between indexing and search. However, if you wish to define specific CPU usage, you will need to do so from `config.yaml`.
+
+This version introduces a `optimizer_cpu_budget` parameter to control the maximum number of CPUs used for indexing. 
+
+> Read more about `config.yaml` in the [Configuration document](https://qdrant.tech/documentation/guides/configuration/).
+
+```yaml
+    optimizer_cpu_budget: 0
+   # CPU budget, how many CPUs (threads) to allocate for an optimization job.
+```
+- If left at 0, Qdrant will keep 1 or more CPUs unallocated - depending on CPU size.
+- If the setting is positive, Qdrant will use this exact number of CPUs for indexing.
+- If the setting is negative, Qdrant will subtract this number of CPUs from the available CPUs for indexing.
+
+For most users, the default `optimizer_cpu_budget` setting will work well. We only recommend you use this if your indexing load is significant.
+
+![CPU management](/blog/qdrant-1.8.x/cpu-management.png)
```

Comment:
I don't think this figure correctly illustrates how this would work.

To me the figure shows that setting some positive number allocates more indexing versus search threads, but that's definitely not the case. Also, the default balance is a lot more on the right, versus the center.

In fact, the meaning of 4 will be highly dependent on the CPU size. On big CPUs setting 4 will significantly impact indexing performance in a bad way.

We could explicitly state the number of CPU cores in this example, and show a few of them, to better illustrate how it works.

But still, the figure might be more confusing than helpful. And then I'd rather vote to remove it instead.